internal/lookout/repository/sql_repository.go

```diff
@@ -70,6 +73,7 @@ type JobRow struct {
 	Cancelled pq.NullTime     `db:"cancelled"`
 	JobJson   sql.NullString  `db:"job"`
 	RunId     sql.NullString  `db:"run_id"`
+	PodNumber sql.NullInt64   `db:"pod_number"`
```

Comment:
could be `sql.NullInt32`

Reply:
`NullInt32` was introduced in golang 1.13, will update as soon as we start using newer go.